NitroxModel/Packets/KnownTechEntry.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class KnownTechEntryAdd : Packet
+    {
+        public TechType TechType;
+        public bool Verbose;
+
+        public KnownTechEntryAdd(TechType techType, bool verbose)
+        {
+            TechType = techType;
+            Verbose = verbose;
+        }
+        public override string ToString()
+        {
+            return "[KnownTechEntryAdd - TechType: " + TechType + " Verbose: " + Verbose + "]";
+        }
+    }
+
+    public class KnownTechEntryChanged : Packet
```

Comment:
It looks like these classes have not be used yet?  If so, please remove them until they are used.